Assets/MixedRealityToolkit.Providers/WindowsMixedReality/WindowsMixedRealityDeviceManager.cs

```diff
@@ -652,6 +652,12 @@ private void RemoveController(InteractionSource interactionSource)
             {
                 IMixedRealityInputSystem inputSystem = Service as IMixedRealityInputSystem;
 
+                if (controller.Visualizer != null &&
```

Comment:
Not exactly sure what kind of tests one would add, but just popping question here...tests? (if applicable) :P

Reply:
I thought about tests! But wasn't successful coming up with any. Most of these changes are only relevant at runtime on-device (the only place the `WindowsMixedRealityDeviceManager` will get a valid source detected), but I'll take a look and see if there's anything I can add.